nixos.nix

```diff
@@ -13,7 +13,8 @@ let
   cfg = config.environment.persistence;
   users = config.users.users;
   allPersistentStoragePaths = zipAttrsWith (_name: flatten) (attrValues cfg);
```

Comment:
I imagine the way we actually want to fix this is to change this call so that it produces the empty attrs we want.

Reply:
Something like this perhaps?
```Nix
{ files = [ ]; directories = [ ]; users = [ ]; } // (zipAttrsWith (_name: flatten) (attrValues cfg))
```

Reply:
Yeah, I like this better